language/Dutch/strings.po

```diff
@@ -165,7 +165,7 @@ msgstr "Selecteer"
 
 #: src/frontend/mame/ui/dirmenu.cpp:58
 msgid "GameOver"
-msgstr "Game over"
+msgstr "Gameover"
```

Comment:
"Game over" was correct

Reply:
strings.po contains seperate strings "Game Over" and "GameOver". Are you sure this ends up correctly?

Reply:
I don't know, ok leave it at Gameover.